templates/administrator/administration/rooms.html.twig

```diff
@@ -0,0 +1,12 @@
+{% extends '../../base.html.twig' %}
```

Comment:
jeśli dobrze pamiętam, to to wywali wyjątek, trzeba sprawdzić, bo nie jestem pewny.

Reply:
https://stackoverflow.com/a/32358776

Reply:
https://stackoverflow.com/a/32358776